lib/twilio/connection.ts

```diff
@@ -1601,6 +1602,12 @@ namespace Connection {
      */
     dscp?: boolean;
 
+    /**
+     * Experimental feature.
+     * Whether to use ICE Aggressive nomination.
+     */
+    enableIceAggressiveNomination?: boolean;
```

Comment:
Is there an IDL we could add this to for Voice? I guess for Video I would consider adding it to our existing ConnectOptions / IceOptions API. 

Naming suggestion, since this flag is describing a workaround that might stop being effective in the future:

```suggestion
    forceAggressiveIceNomination?: boolean;
```

Reply:
Sounds good on the name suggestion. I don't think we have IDL for voice js :(